### ISSUE TITLE
training problem about loss

### ISSUE THREAD

**Issue by renrenzsbbb** [Doubiiu/CodeTalker#79]
Thanks for your great work and open code. I use your code to train my customer dataset, and find that the validation loss is twice as much as training loss . can you tell me whether you are meet with the propblem and give me some advice if possiable. Thanks in advance.
